src/plugin.js

```diff
@@ -230,10 +254,12 @@ const contribAdsPlugin = function(options) {
           this.state = 'ads-ready';
         },
         play() {
```

Comment:
@incompl , from what I can tell in testing, the only time we are getting a play event while in the content-set state is when initially loading and autoplaying a video. I tried everything I can with playlists and play always comes when we are in the ads-ready state....